src/legacy/smartweave-global.ts

```diff
@@ -257,7 +257,10 @@ class Vrf {
 export class KV {
   private _kvBatch: BatchDBOp<any>[] = [];
 
-  constructor(private readonly _storage: SortKeyCache<any> | null, private readonly _transaction: Transaction) {}
+  constructor(
```

Comment:
hmm, is this our standard linter formatting?

Reply:
Nope, I done my self, because parameters were not fitting into my wide 13inch screen
However I have ran linter - ohhh i see we have also prettier :) Now everything should be properly formatted :D